internal/pkg/service/stream/storage/level/local/encoding/config.go

```diff
@@ -0,0 +1,39 @@
+package encoding
+
```

Comment:
- The `encoding` package contains the part of the write pipeline that runs on the (HTTP) Source node.
- The code was moved here from the `writer`/`diskwriter` pkg.